.github/ISSUE_TEMPLATE/patternfly-issue-template.md

```diff
@@ -0,0 +1,17 @@
+---
+name: PatternFly issue template
+about: Please populate the PatternFly issue template so we can learn more about your
+  needs.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+### Describe the issue. What are the expected and unexpected behavior categories?
+
+### Please provide the steps to reproduce so we can better understand and address the issue. Feel free to link CodePen or another tool.
+
+### Would you consider this issue a bug or enhancement?
+
+### What is your product and what release version are you targeting?
```

Comment:
Could also ask if this is blocking product work.

Reply:
What do you think about this feedback @tlabaj ?

Reply:
I think the wording here is what the team agreed upon.  This give us more insight into an actual timeline for when the feature is needed. 
cc @LHinson